CityOfDetroit/bloom
# 1234/minor copy updates

## Issue

- Addresses #1234 

## Description

This PR updates the content to reflect rows 5, 8, 9, and 10 in the following sheet: https://docs.google.com/spreadsheets/d/1KmIs93OKQIDsKyay90MLWmItLVBCyAytJJ8s5p-_Zm8/edit#gid=270098974

The only other changes were to remove the link to the HMI table section since it seemed redundant with the new wording.

## How Can This Be Tested/Reviewed?
Select a property with the "Confirmed by Property" badge. Then from the individual property page check for the following:
Confirmed Tooltip content: This can be tested by hovering over the "Verified by Property" badge.
AMI Tooltip content: This can be tested by hovering over the small info icon next to 'AMI" on the unit type table.
*Note* The content below is only visible with unit type charts that have full AMI information. 
Unit Type text: This can be tested by reading underneath the unit type table.
HMI: This can be tested by reading the text underneath the "1 | Household Maximum Income" header.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
